web/src/components/tasks/business-formation/contacts/Signatures.test.tsx

```diff
@@ -250,24 +259,62 @@ describe("Formation - Signatures", () => {
     );
 
     it.each([...legalStructureIds])(
-      "displays inline error when name and type is missing for %s",
+      "displays inline error when name, signature, and type is missing when legalStructure is %s",
       async (legalStructureId) => {
         const page = await getPageHelper({ businessPersona: "FOREIGN", legalStructureId }, { signers: [] });
 
         await attemptApiSubmission(page);
-        const singerNameErrorType = (): HTMLElement | null => {
-          return screen.queryByText(Config.formation.fields.signers.errorInlineSignerName);
+        const singerNameAndSignatureErrorType = (): HTMLElement | null => {
+          return screen.queryByText(Config.formation.fields.signers.errorInlineNameAndSignature);
         };
         const signerTitleErrorType = (): HTMLElement | null => {
           return screen.queryByText(Config.formation.fields.signers.errorInlineSignerTitle);
         };
-        expect(singerNameErrorType()).toBeInTheDocument();
+        expect(singerNameAndSignatureErrorType()).toBeInTheDocument();
         expect(signerTitleErrorType()).toBeInTheDocument();
       }
     );
```

Comment:
This appears to be the same test above, lines 171-200.

Reply:
They are testing different things, the one above is testing that the 'greater than 50 characters' error shows up in line and this is testing that name and signature  is missing and add title error shows up, they are also for different users, poppy vs dakota nexus